### DISCUSSION TITLE
a GUI question...

### DISCUSSION THREAD

**Discussion by madjestic** [ivanperez-keera/dunai#417] [Ideas]
Oh boy, this might be a big one...

We all know that GUIs are a lot of work.  As far as I know, FRP libraries like Yampa or Dunai are not compatible with GUI libs like dear-imgui.hs, correct me if I am wrong - essentially because they have their own update models.  What would be a recommended course of action here, is there an easier path than writing your own GUI system from scratch?

Cheers,
Vlad.

**Comment by ivanperez-keera** [comment]
Here are a few pointers:

- https://github.com/ivanperez-keera/Yampa/discussions/264
- https://github.com/ivanperez-keera/dunai/tree/develop/dunai-examples/keerahails
- https://github.com/keera-studios/keera-hails/tree/develop/keera-hails-reactive-yampa
